# exception
delivery-result.already.exists.exception=이미 존재합니다
delivery-result.not.found.exception=찾을 수 없습니다
delivery.already.exists.exception=이미 존재합니다
delivery.not.found.exception=찾을 수 없습니다
delivery-subject.not.found.exception=찾을 수 없습니다
